jupyter/restore_raft_quorum_with_tablets.ipynb

```diff
@@ -0,0 +1,680 @@
+   "source": [
+    "## Step 6. \n",
+    "Find recovery coordinator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a5c9943",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = verification_node.run_cqlsh(\"select value from system.scylla_local where key = 'raft_group0_id'\", split=True)\n",
+    "group0_id = result[3]\n",
+    "commit_idx = 0\n",
+    "recovery_coordinator = None\n",
+    "for node in base_alive_nodes:\n",
+    "    cidx = verification_node.run_cqlsh(f\"select commit_idx from system.raft where group_id = {group0_id}\", split=True)[3]\n",
+    "    if int(cidx) > commit_idx:\n",
+    "        commit_idx = int(cidx)\n",
+    "        recovery_coordinator = node\n",
+    "\n",
+    "print(recovery_coordinator.name, recovery_coordinator.ip_address, recovery_coordinator.host_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f076d21c",
+   "metadata": {},
+   "source": [
+    "## Step 7. \n",
+    "Remove raft_group_id from system.scylla_local and truncate system.discovery on each live node."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c8156024",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for node in base_alive_nodes:\n",
+    "    node.run_cqlsh(\"DELETE value FROM system.scylla_local WHERE key = 'raft_group0_id'\")\n",
+    "    node.run_cqlsh(\"TRUNCATE system.discovery\")\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ed22247",
+   "metadata": {},
+   "source": [
+    "## Step 8.\n",
+    "Set the new scylla.yaml parameter, recovery_leader, to Host ID of the recovery leader on each live node"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95cc628e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "recovery_host_id = recovery_coordinator.host_id\n",
+    "for node in base_alive_nodes:\n",
+    "    node.remoter.sudo(f\"\"\"echo 'recovery_leader: {recovery_host_id}' | sudo tee -a /etc/scylla/scylla.yaml\"\"\")\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "08f23e58",
+   "metadata": {},
+   "source": [
+    "## Step 9.\n",
+    "Rolling restart all live nodes, but the recovery leader must be restarted first."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "545d32ff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "cluster.restart_scylla(nodes=[recovery_coordinator])\n",
+    "cluster.restart_scylla(nodes=[node for node in base_alive_nodes if node != recovery_coordinator])\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4691849",
+   "metadata": {},
+   "source": [
+    "## Step 10.\n",
+    "Remove/replace all dead nodes\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "617dc05e",
+   "metadata": {},
+   "source": [
+    "### step 10.1\n",
+    "Remove stopped node in alive dc. it is possible because each rack had 2 nodes"
```

Comment:
If you revert stopping one node from alive DC, you will have to remove this step. See my comments above.

The same goes for step 10.4.